tests/testdata/compile/node_modules_symlink_outside/main_compile_file.out

```diff
@@ -1,5 +1,4 @@
 Compile file:///[WILDCARD]/node_modules_symlink_outside/main.ts to [WILDCARD]
-Warning Symlink target is outside '[WILDCARD]node_modules_symlink_outside'. Inlining symlink at '[WILDCARD]node_modules_symlink_outside[WILDCARD]node_modules[WILDCARD]test.txt' to '[WILDCARD]target.txt' as file.
```

Comment:
The root of the VFS is determined after everything is inside it, so this is now represented properly inside it.